css/book-table.css

```diff
@@ -85,6 +82,100 @@
   border: 2px solid red;
 }
 
+/* ADDING RESPONSIVNESS */
+
+/* FOR SCREENS < 1100 PX */
+
+@media (max-width: 1100px) {
+  .booking-page-content {
+    margin: 150px auto;
+  }
+
+  .booking-form {
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(7, auto);
+    row-gap: 20px;
+    padding: 2rem;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .booking-form__heading {
+    grid-area: 1 / 1 / 2 / 2;
+    margin-bottom: 30px;
+    text-align: center;
+  }
+
+  .first-name-container {
+    grid-area: 2 / 1 / 3 / 2;
+  }
+
+  .last-name-container {
+    grid-area: 3 / 1 / 4 / 2;
+  }
+
+  .email-container {
+    grid-area: 4 / 1 / 5 / 2;
+  }
+
+  .phone-container {
+    grid-area: 5 / 1 / 6 / 2;
+  }
+
+  .notes-container {
+    grid-area: 6 / 1 / 7 / 2;
+  }
+
+  .booking-form__a {
+    grid-area: 7 / 1 / 8 / 2;
+  }
+
+  .booking-form__btn {
+    max-width: 400px;
+  }
+
+  .booking-form__input {
+    min-width: 500px;
+  }
+}
+
+/* FOR SCREENS < 600 PX */
+@media (max-width: 600px) {
+  .booking-form {
+    margin: 0 50px;
+  }
+
+  .booking-form__input {
+    min-width: 400px;
+  }
+}
+
+/* FOR SCREENS < 480 PX */
+@media (max-width: 480px) {
+  .booking-page-content {
+    background-color: #00ffff;
```

Comment:
You might not need this background-color value since the form takes the whole space at this breakpoint and the main container is not visible.

Reply:
you are right, removed, thanks!